docs/generated/eventlog.md

```diff
@@ -2465,6 +2465,13 @@ contains common SQL event/execution details.
 | `TransactionID` | Transaction ID of the query. | no |
 | `DatabaseID` | Database ID of the query. | no |
 | `StatementFingerprintID` | Statement fingerprint ID of the query. | no |
+| `MaxFullScanRowsEstimate` | Maximum number of rows scanned by a full scan, as estimated by the optimizer. | no |
```

Comment:
This is the maximum full scan over both full **index** and full **table** scans correct?